templates/azure/terraform/schemas/datetime_and_duration.erb

```diff
@@ -0,0 +1,45 @@
+"<%= property.name.underscore -%>": {
+    Type: schema.TypeString,
+<%  if property.default_from_api && data_source_input.empty? -%>
+    Computed: true,
+    Optional: true,
+<%     if property.schema_config_mode_attr -%>
+    ConfigMode: schema.SchemaConfigModeAttr,
+<%     end -%>
+<%   elsif property.required && (data_source_input.empty? || data_source_input.include?(property)) -%>
+    Required: true,
+<%   elsif property.output || (!data_source_input.empty? && !data_source_input.include?(property)) -%>
+    Computed: true,
+<%   else -%>
+    Optional: true,
+<%   end -%>
+<%   if property.deprecated? -%>
+    Deprecated: "<%= property.deprecation_message %>",
+<%   end -%>
+<%   if force_new?(property, object) && data_source_input.empty? -%>
+    ForceNew: true,
+<%   end -%>
+<%   if property.validation.nil? && !property.output -%>
+<%     if property.is_a?(Api::Azure::Type::ISO8601DateTime) -%>
+    ValidateFunc: validateRFC3339Date,
```

Comment:
*Discussion* If the only difference between `ISO8601` and `String` is `ValidateFunc`, we can modify it in `primitive.erb` directly.

Reply:
Not really. There are also difference in the `set` function and expand/flatten methods.

Reply:
Now, it was moved into `primitive.erb`.